src/annotator/components/NotebookModal.js

```diff
@@ -48,6 +48,10 @@ function NotebookIframe({ config, groupId }) {
  * @param {NotebookModalProps} props
  */
 export default function NotebookModal({ eventBus, config }) {
+  // Temporal solution: while there is no mechanism to sync new annotations in
+  // the notebook, we force re-rendering of the iframe on every 'openNotebook'
+  // event so we fetch the new annotations.
+  const [iframeKey, setIframeKey] = useState(Math.random());
```

Comment:
I think we can just make this a super simple counter and just increment on change

`useState(0);`